### DISCUSSION TITLE
我想让其记住《甄嬛传》76 集里面的所有剧情和画面呢？

### DISCUSSION THREAD

**Discussion by ponponon** [zai-org/CogVLM2#198] [Q&A]
假设在使用上面的 vlm 的时候。我想让其记住《甄嬛传》76 集里面的所有剧情和画面呢？

怎么添加训练？有思路可以分享吗？谢谢

训练之后，我希望可以实现的效果：

- 问它第几集几分几秒再讲什么，可以告诉我对应的剧情
- 给它一个图片，可以告诉我出现在第几集的几分几秒

**Comment by huangshiyu13** [comment]
构造相应的视觉和问答对就行了

**Reply by ponponon** [reply]
> 构造相应的视觉和问答对就行了

做微调是吗？

**Reply by huangshiyu13** [reply]
是的
